autopush/web/base.py

```diff
@@ -238,7 +238,10 @@ def _response_err(self, fail):
         running.
 
         """
+        from twisted.internet.error import ConnectionDone
         fmt = fail.value.message or 'Exception'
+        if fail.value.__class__ in [ConnectionDone]:
```

Comment:
why not isinstance?

Reply:
wasn't sure if there were going to be others.

Reply:
commonly unknown but so nice: isinstance arg 2 can be a tuple of types